src/pykiso/lib/connectors/cc_serial.py

```diff
@@ -0,0 +1,165 @@
+##########################################################################
```

Comment:
Regarding the discussion we had about the fix in pykiso-tags, the fix has to be added here: https://github.com/eclipse/kiso-testing/blob/master/src/pykiso/tool/show_tag.py#L287

And simply is: `output = Path(output)`

Reply:
added